integration-ddb-to-lambda/example.rb

```diff
@@ -0,0 +1,18 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+def lambda_handler(event:, context:)
+    return 'received empty event' if event['Records'].empty?
+  
+    event['Records'].each do |record|
+      log_dynamodb_record(record)
+    end
+  
+    "Records processed: #{event['Records'].length}"
```

Comment:
The other snippets in this suite log the entire JSON record. Can we modify this to do that as well?

Reply:
Updated the code, please see the new commit.